src/rpc/Counters.cpp

```diff
@@ -69,27 +65,26 @@ Counters::rpcComplete(std::string const& method, std::chrono::microseconds const
 void
 Counters::rpcForwarded(std::string const& method)
 {
-    if (!validHandler(method))
```

Comment:
The method check is after the forward checking. If user set "ledger_index" : current, it will forward to rippled regardless the method is valid or not.

Reply:
I guess that's ok, wdyt? it may result in creating extra methods in the counters however. maybe i can add this check on the call site instead.